var/spack/repos/builtin/packages/py-awkward/package.py

```diff
@@ -25,12 +30,13 @@ class PyAwkward(PythonPackage):
     patch('pybind11.patch', when="@:1.2.2")
     patch('pybind11_02.patch', when="@1.2.3:")
 
-    depends_on('py-setuptools', type='build')
+    depends_on('py-setuptools@42.0:', type='build')
     depends_on('py-pyyaml', type='build')
 
     depends_on('python@2.7:2.8,3.5:', type=('build', 'run'))
     depends_on('py-numpy@1.13.1:', type=('build', 'run'))
     depends_on('py-pybind11', type=('build', 'link'))
     depends_on('dlpack', when="@1.0.0:")
     depends_on('rapidjson')
-    depends_on('cmake', type='build')
+    depends_on('cmake@3.13:', type='build')
```

Comment:
I think this should be py-cmake

Reply:
Potentially, but I haven't had any problems building this with "standalone" cmake. I am not entirely sure if that just happened to work, or if that worked as intended.

Reply:
We currently pass `--no-deps` to the `pip install` call which disables dependency checking. That's the only reason the current solution works. I think it would be better to explicitly add the dependency.

Reply:
Ah, I see. I wasn't aware of that. I can build with `py-cmake` on a centos7 machine, but I run into some problems when trying to do the same thing on my local Ubuntu, but that might be unrelated, see #29789 for a few more details.

Reply:
In that case I'm fine with leaving it as is for now.